Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -41,15 +46,22 @@ + (instancetype)sharedManager;
 
 @implementation PINAnimatedImageManager
 
-+ (void)initialize
++ (void)load
```

Comment:
Usually better practice for a 3rd party library to use `initialize` just because it's only called as soon as the first message to the class is sent not when the class is loaded at launch.